test-series/1/test-1.0.0.ipynb

```diff
@@ -0,0 +1,1449 @@
+{
```

Comment:
<p>How is the EoR amplified with frequency?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera-validation/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Reply:
```
rescaling = 10*(100e6/master_uvd.freq_array[0])**3
master_uvd.data_array *= rescaling[None, None, :, None]
```
So the EoR visibilities are amplified overall by a factor of 10, and have a frequency dependence (f_0/f)^3, with f_0 = 100 MHz.